src/localisation-service.ts

```diff
@@ -127,6 +128,14 @@ export function formatDateLong(value: Date): string {
  * @returns
  */
 export function formatTime(value: Date): string {
+  const iDeviceOSVersion = parseFloat(
+    parser(window.navigator.userAgent).os.version || '',
+  )
+  if (iDeviceOSVersion < 13.0) {
+    const time = tenants.current.intlFormats.olderIOSTime.format(value)
+    const newTime = time.split(':')
```

Comment:
Im not sure if its possible, but could we use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/formatToParts instead of splitting the string?